inc/ticket.class.php

```diff
@@ -304,14 +306,35 @@ static function addHistoryOnAddGroup(CommonDBTM $item) {
          $group->getFromDB($groups_id);
 
          $task = new TicketTask();
+          $content = __("Escalation to the group", "escalade") . " " . $group->getName();
+          if (isset($item) && isset($item->input['escalade_comment'])) {
+              $content .= "<br><strong>" . __('User comment', 'escalade') . " :" . "</strong><br>";
+              $content .= \Glpi\RichText\RichText::getTextFromHtml($item->input['escalade_comment']);
+          }
+
          $task->add([
             'tickets_id' => $tickets_id,
             'is_private' => true,
             'state'      => Planning::INFO,
-            'content'    => Toolbox::addslashes_deep(__("escalated to the group", "escalade") . " " . $group->getName())
+            'content'    => Toolbox::addslashes_deep($content)
          ]);
       }
 
+       if (!$_SESSION['plugins']['escalade']['config']['task_history']) {
```

Comment:
Same as previous comment but this time the code is also executed despite the `task_history` being disabled.